frontend/react-neko-chat/src/styles.css

```
@@ -2569,14 +2615,18 @@ body.electron-chat-window.subtitle-web-host .compact-chat-surface-shell {
+    var(--compact-chat-surface-edge-top)
+    var(--compact-chat-surface-edge-right)
+    var(--compact-chat-surface-edge-bottom)
+    var(--compact-chat-surface-edge-left);
   border-radius: 999px;
-  background: rgba(255, 255, 255, 0.08);
-  box-shadow:
-    0 0 0 1px rgba(208, 233, 255, 0.14),
-    0 6px 18px rgba(42, 89, 132, 0.05);
-  backdrop-filter: blur(18px) saturate(1.22) brightness(1.08);
-  -webkit-backdrop-filter: blur(18px) saturate(1.22) brightness(1.08);
+  background: rgba(255, 255, 255, 0.035);
```

Stylelint (17.14.0)

[error] Overridden property "background-clip" by shorthand "background" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)